app/graphql/types/customer_metadata/input.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Types
+  module CustomerMetadata
+    class Input < Types::BaseInputObject
+      graphql_name 'CustomerMetadataInput'
+
+      argument :id, ID, required: false
+      argument :key, String, required: true
+      argument :value, String, required: true
+
+      argument :display_in_invoice, Boolean, required: true
```

Comment:
Is `display_on_invoice` wording not better?

Reply:
actually I have checked those two versions; both are grammatically correct but from what I have seen syntax `in invoice` is used more 😅